llvm/test/Transforms/LoopVectorize/RISCV/vplan-vp-select-intrinsics.ll

```diff
@@ -0,0 +1,101 @@
+; REQUIRES: asserts
```

Comment:
Need to precommit a test in a separate patch

Reply:
spilt done